site/content/contribute/desktop/dependencies.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "Dependencies"
+heading: "Managing Dependencies in the Desktop App"
+description: "Working with other libraries in the Desktop App"
+date: 2019-01-22T00:00:00-05:00
+weight: 3
+---
+
+The Desktop App uses `npm` to manage its dependencies.
+
+We usually try to keep each major dependency version locked such that we don't accidentally introduce any bugs or breaking changes by upgrading.
+
+All dependencies are locked using a `package-lock.json` file to ensure that we don't change the package versions used to build unless explicitly upgrading the package. Thus if a PR contains changes to `package-lock.json` without explicitly changing a dependency, we will usually ask the contributor to revert those changes.
```

Comment:
What are the criteria for selecting changes to dependencies? These instructions kinda make it sound like they should never be changed.